08.Third-party-API-implementation.md

```diff
@@ -0,0 +1,12 @@
+# 3rd Party API Implementation
+
+Some contributors are creating and maintaining external libraries that could simplify your life on your project. Those contributors created packages that use Gitter API in different shapes (progamming languages, main goals, ...). You can retrieve a list of the current existing packages below, sorted by languages.
+
+## .NET (C#)
+
+* [Gitter#](https://github.com/Odonno/gitter-api-pcl) - provides you everything you need to use Gitter API (in both usual and Rx ways) on Windows (Phone) apps, .NET Framework and Xamarin platforms.
+* [Gitter# Auth](https://github.com/Odonno/gitter-api-auth) - provides you a way to retrieve Token through OAuth2 process.
```

Comment:
I think it should be a sub-bullet. I understand the visual aspect being that there is only two things under this section.